rocAL/rocAL/source/augmentations/color_augmentations/node_brightness.cpp

```diff
@@ -24,30 +24,22 @@ THE SOFTWARE.
 #include "node_brightness.h"
 #include "exception.h"
 
-
 BrightnessNode::BrightnessNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
         Node(inputs, outputs),
         _alpha(ALPHA_RANGE[0], ALPHA_RANGE[1]),
-        _beta (BETA_RANGE[0], BETA_RANGE[1]) { }
+        _beta (BETA_RANGE[0], BETA_RANGE[1]) {}
 
 void BrightnessNode::create_node() {
     if(_node)
         return;
 
-    _alpha.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
-    _beta.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
-
-    int input_layout = (int)_inputs[0]->info().layout();
-    int output_layout = (int)_outputs[0]->info().layout();
-    int roi_type = (int)_inputs[0]->info().roi_type();
-    vx_scalar in_layout_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &input_layout);
-    vx_scalar out_layout_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &output_layout);
-    vx_scalar roi_type_vx = vxCreateScalar(vxGetContext((vx_reference)_graph->get()), VX_TYPE_INT32, &roi_type);
-    _node = vxExtrppNode_Brightness(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), in_layout_vx, out_layout_vx, roi_type_vx);
+    _alpha.create_array(_graph, VX_TYPE_FLOAT32, _batch_size);
+    _beta.create_array(_graph, VX_TYPE_FLOAT32, _batch_size);
+    _node = vxExtRppBrightness(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), _input_layout, _output_layout, _roi_type);
```

Comment:
please have the layout extracted from the tensor class instead of having another variables for it. Plus here the node can take a vector of tensors

Reply:
Added changes to extract the input layout, output layout and roi type from the input and output tensor in every augmentations node class instead of having it common in Node class